scanner/cmd/scannerctl/main.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/spf13/cobra"
 	"github.com/stackrox/rox/pkg/mtls"
+	"github.com/stackrox/rox/pkg/scanners/scannerv4/client"
```

Comment:
I just realized that I forgot about `scannerctl`. It's quite ugly to have the scanner command line dependent on `rox/pkg/scanners`, which contains a lot of code and packages related to image integrations. Is it possible to move this to a different location under `rox/pkg/`? I think `pkg/scannerv4/client` would be a good option. Alternatively, `pkg/scanner/client` could also work, but having `pkg/scanners`, `pkg/scanner`, and `pkg/scan` is confusing, so `pkg/scannerv4` might be the best choice.

Reply:
Alrighty, will put it under `pkg/scannerv4` 

Perhaps this is a style choice, but I believe its cleaner to have a single `pkg/scanners` directory to hold all 're-usable stuff for scanners'  - the `scannersv4/client` subdir doesn't necessarily mean it's associated with an integration.